app/jobs/reports/drop_off_report.rb

```diff
@@ -8,6 +8,8 @@ class DropOffReport < BaseReport
 
     attr_accessor :report_date
 
+    # Generate a drop off report for the week including the passed timestamp
+    # @param [DateTime]
```

Comment:
if this is `Time.zone.yesterday.end_of_day`, it's an `ActiveSupport::TimeWithZone` which is more like a `Time` object than a `DateTime` object IMO

Reply:
Either should be acceptable. I suppose something like `[DateTime|Time]` would be more accurate?